test/infrastructure/docker/hack/verify-spelling.sh

```diff
@@ -38,7 +38,7 @@ URL="https://github.com/client9/misspell"
 echo "Cloning ${URL} in ${TMP_DIR}..."
 git clone --quiet --depth=1 "${URL}" "${TMP_DIR}"
 pushd "${TMP_DIR}" > /dev/null
-go mod init
+go mod init github.com/client9/misspell
```

Comment:
we should use hack/tools/tools.go for this

Reply:
github.com/client9/misspell doesn't use go modules and not seeing a way around that other than using this script.

Reply:
This script has been deleted in favor of the spelling linter.